config/prow/cluster/monitoring/mixins/prometheus/ghproxy_alerts.libsonnet

```diff
@@ -9,7 +9,7 @@
           {
             alert: 'ghproxy-specific-status-code-abnormal',
             expr: |||
-              sum(rate(github_request_duration_count{status=~"[45]..",status!="404",status!="410"}[5m])) by (status,path) / ignoring(status) group_left sum(rate(github_request_duration_count[5m])) by (path) * 100 > 10
+              sum(rate(github_request_duration_count{status=~"[45]..",status!="404",status!="410"}[30m])) by (status,path) / ignoring(status) group_left sum(rate(github_request_duration_count[5m])) by (path) * 100 > 10
```

Comment:
I skimmed through some of the #prow-alerts channel and didn't see any 5xx alerts.  WDYT about keeping the tighter alerting window for 5xx and only loosening 4xx?
5mins to 30mins is a pretty big change and delays the alerts for so long that they we'd be likely to notice serious problems ourselves before the alert fires. Could we choose a smaller time window if we also increased the error threshold to more than 10%?

Reply:
good point about 5xx, updated to keep 5xx still alert with 5m window events.

I have seen 100% for example https://kubernetes.slack.com/archives/CQP3RC68Y/p1614920511052900, which imo would be pretty hard to brought down to 20% even with doubled time, so I think diluting it out is more effective, still finger crossed on this though